--- a/net/neoforged/neoforge/network/payload/KnownRegistryDataMapsReplyPayload.java
+++ b/net/neoforged/neoforge/network/payload/KnownRegistryDataMapsReplyPayload.java
@@ -22,7 +22,7 @@
 @ApiStatus.Internal
 public record KnownRegistryDataMapsReplyPayload(
         Map<ResourceKey<? extends Registry<?>>, Collection<Identifier>> dataMaps) implements CustomPacketPayload {
-    public static final Type<KnownRegistryDataMapsReplyPayload> TYPE = new Type<>(Identifier.fromNamespaceAndPath("neoforge", "known_registry_data_maps_reply"));
+    public static final Type<KnownRegistryDataMapsReplyPayload> TYPE = new Type<>(Identifier.fromNamespaceAndPath(fuzs.multiloaderdataextensions.common.impl.MultiloaderDataExtensions.MOD_ID, "known_registry_data_maps_reply"));
     public static final StreamCodec<FriendlyByteBuf, KnownRegistryDataMapsReplyPayload> STREAM_CODEC = StreamCodec.composite(
             ByteBufCodecs.map(
                     Maps::newHashMapWithExpectedSize,
