--- a/registries/datamaps/DataMapFile.java
+++ b/registries/datamaps/DataMapFile.java
@@ -15,13 +15,11 @@
 import net.minecraft.resources.ResourceKey;
 import net.minecraft.tags.TagKey;
 import net.minecraft.util.ExtraCodecs;
-import net.neoforged.neoforge.common.conditions.ConditionalOps;
-import net.neoforged.neoforge.common.conditions.WithConditions;
 import net.neoforged.neoforge.common.util.NeoForgeExtraCodecs;
 
 public record DataMapFile<T, R>(
         boolean replace,
-        Map<Either<TagKey<R>, ResourceKey<R>>, Optional<WithConditions<DataMapEntry<T>>>> values,
+        Map<Either<TagKey<R>, ResourceKey<R>>, DataMapEntry<T>> values,
         List<DataMapEntry.Removal<T, R>> removals) {
     public static <T, R> Codec<DataMapFile<T, R>> codec(ResourceKey<Registry<R>> registryKey, DataMapType<R, T> dataMap) {
         final Codec<Either<TagKey<R>, ResourceKey<R>>> tagOrValue = ExtraCodecs.TAG_OR_ELEMENT_ID.xmap(
@@ -46,7 +44,7 @@
 
         return RecordCodecBuilder.create(in -> in.group(
                 Codec.BOOL.optionalFieldOf("replace", false).forGetter(DataMapFile::replace),
-                ExtraCodecs.strictUnboundedMap(tagOrValue, ConditionalOps.createConditionalCodecWithConditions(DataMapEntry.codec(dataMap))).fieldOf("values").forGetter(DataMapFile::values),
+                ExtraCodecs.strictUnboundedMap(tagOrValue, DataMapEntry.codec(dataMap)).fieldOf("values").forGetter(DataMapFile::values),
                 removalsCodec.optionalFieldOf("remove", List.of()).forGetter(DataMapFile::removals))
                 .apply(in, DataMapFile::new));
     }
