--- a/network/payload/RegistryDataMapSyncPayload.java
+++ b/network/payload/RegistryDataMapSyncPayload.java
@@ -29,26 +29,26 @@
 @SuppressWarnings({ "unchecked", "rawtypes" })
 public record RegistryDataMapSyncPayload<T>(ResourceKey<? extends Registry<T>> registryKey,
         Map<Identifier, Map<ResourceKey<T>, ?>> dataMaps) implements CustomPacketPayload {
-    public static final CustomPacketPayload.Type<RegistryDataMapSyncPayload<?>> TYPE = new Type<>(Identifier.fromNamespaceAndPath("neoforge", "registry_data_map_sync"));
+    public static final CustomPacketPayload.Type<RegistryDataMapSyncPayload<?>> TYPE = new Type<>(Identifier.fromNamespaceAndPath(fuzs.multiloaderdataextensions.common.impl.MultiloaderDataExtensions.MOD_ID, "registry_data_map_sync"));
     public static final StreamCodec<RegistryFriendlyByteBuf, RegistryDataMapSyncPayload<?>> STREAM_CODEC = StreamCodec.ofMember(
             RegistryDataMapSyncPayload::write, RegistryDataMapSyncPayload::decode);
 
     public static <T> RegistryDataMapSyncPayload<T> decode(RegistryFriendlyByteBuf buf) {
         //noinspection RedundantCast javac complains about this cast
         final ResourceKey<Registry<T>> registryKey = (ResourceKey<Registry<T>>) (Object) buf.readRegistryKey();
-        final Map<Identifier, Map<ResourceKey<T>, ?>> attach = buf.readMap(FriendlyByteBuf::readIdentifier, (b1, key) -> {
+        final Map<Identifier, Map<ResourceKey<T>, ?>> attach = fuzs.multiloaderdataextensions.fabric.impl.network.FriendlyByteBufHelper.readMap(buf, FriendlyByteBuf::readIdentifier, (b1, key) -> {
             final DataMapType<T, ?> dataMap = RegistryManager.getDataMap(registryKey, key);
-            return b1.readMap(bf -> bf.readResourceKey(registryKey), (bf, _) -> readJsonWithRegistryCodec((RegistryFriendlyByteBuf) bf, dataMap.networkCodec()));
+            return fuzs.multiloaderdataextensions.fabric.impl.network.FriendlyByteBufHelper.readMap(b1, bf -> bf.readResourceKey(registryKey), (bf, _) -> readJsonWithRegistryCodec((RegistryFriendlyByteBuf) bf, dataMap.networkCodec()));
         });
         return new RegistryDataMapSyncPayload<>(registryKey, attach);
     }
 
     public void write(RegistryFriendlyByteBuf buf) {
         buf.writeResourceKey(registryKey);
-        buf.writeMap(dataMaps, FriendlyByteBuf::writeIdentifier, (b1, key, attach) -> {
+        fuzs.multiloaderdataextensions.fabric.impl.network.FriendlyByteBufHelper.writeMap(buf, dataMaps, FriendlyByteBuf::writeIdentifier, (b1, key, attach) -> {
             final DataMapType<T, ?> dataMap = RegistryManager.getDataMap(registryKey, key);
             // TODO - make datamaps use stream codecs once datapack registries use them too
-            b1.writeMap(attach, FriendlyByteBuf::writeResourceKey, (bf, _, value) -> writeJsonWithRegistryCodec((RegistryFriendlyByteBuf) bf, (Codec) dataMap.networkCodec(), value));
+            fuzs.multiloaderdataextensions.fabric.impl.network.FriendlyByteBufHelper.writeMap(b1, attach, FriendlyByteBuf::writeResourceKey, (bf, _, value) -> writeJsonWithRegistryCodec((RegistryFriendlyByteBuf) bf, (Codec) dataMap.networkCodec(), value));
         });
     }
 
